### PR TITLE
Validators docs

### PR DESCRIPTION
**Pull request by Anitaachu**
#### Did you test the changes locally ?
- [ ] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

**Review by bogdan-rosianu — APPROVED** [review]

File: docs/validators/installation/install-update.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+After preparing the user permissions, the script configurations, and the keys, the actual node installation can begin. The Validator script is a multi-purpose tool for managing your node, it is accessible to all networks: Mainnet, Devnet, and Testnet.
+
+- *Note: This process installs a validator node for all Elrond networks (mainnet, devnet, and testnet).*
+
+Following these few steps, we will work on installing the Elrond Network validator node to get it up and running on your local machine.
+
+ For installation use`1 - install`
+
+```bash
+ ~/elrond-go-scripts-mainnet/script.sh
```
Comment (bogdan-rosianu) [inline docs/validators/installation/install-update.md:21]: `elrond-go-scripts` ?

File: docs/validators/installation/manage-node.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+title: Manage a validator node
+---
+
+Your node will start as an observer. In order to make it into a validator, you will need to have 2500 xEGLD tokens. You can reach out to an admin in our [Telegram community](https://t.me/ElrondValidators) who will gladly help.
+
+Follow these steps to manage your validator node.
+
+Let’s begin!
+
+First, you need to create an Elrond wallet. You can create this wallet on either the [devnet](https://devnet-wallet.elrond.com) or [testnet](https://testnet-wallet.elrond.com).
```
Comment (bogdan-rosianu) [inline docs/validators/installation/manage-node.md:12]: somehow duplicated to what we have in mainnet/testnet/devnet. what's the approach here? delete the other scripts docs? or ?

**Comment by bogdan-rosianu** [comment]
please add the new pages to `sidebars.json`, otherwise, they won't appear in the docs

File: docs/validators/installation/manage-node.md (hunk 2 above)
Comment (Anitaachu) [reply docs/validators/installation/manage-node.md:12]: The aim is to merge what is available in mainnet/testnet/devnet, in order to have just one script.

**Pull request merged**